### DISCUSSION TITLE
recwl location

### DISCUSSION THREAD

**Discussion by xerious** [johanmeijer/grott#564] [Q&A]
Hello,

Does anyone know the location of the 'recwl.txt' file.
it keeps telling me it can't find it.

the code just does: 
 with open('recwl.txt') as f:
                self.recwl = f.read().splitlines()
            if self.verbose : print("\nGrott external record whitelist: 'recwl.txt' read")
            
I have placed the file in config/grott, config and on '/'
But none seems to work

**Comment by johanmeijer** [comment]
For normal processing you do not need this file. It is only there for advanced grott usage/customisation (and for my to test things). 
To be honest I run without this file myself.
